tests/5.1/scope/test_scope_nowait_construct.c

```diff
@@ -0,0 +1,49 @@
+//-------------test_scope_nowait_construct.c-------------//
+//
+//OpenMP API Version 5.1 Aug 2021
+//
+//Tests the behavior of the scope construct with nowait
+//specified. This test should remove the implied barrier
+//creatd by the scope construct when nowait is specified.
+//-------------------------------------------------------//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+#include <unistd.h>
+
+#define N 1024
+
+int test_scope_nowait(){
+	int errors = 0;
+	int test_arr[N];
+	for (int i=0; i<N, i++){
+		test_arr[i] = 1;
+		sum += i;
```

Comment:
```
test_scope_nowait_construct.c:24:17: error: ‘sum’ undeclared (first use in this function)
   24 |                 sum += i;
```

Note: It is declared in Line 35 – i.e. the decl needs to be moved up.

And please decide whether you want to use `sum` – or `test_int`.